.bazelrc

```diff
@@ -76,6 +76,9 @@ test:coverage --spawn_strategy=local
 build --define=versioned=false
 build:versioned --workspace_status_command=tools/buildstamp/get_workspace_status --stamp --define=versioned=true
 
+build:dbg-darwin --config=dbg --platforms=@//tools/config:darwin_x86_64
+build:dbg-linux --config=dbg --platforms=@//tools/config:linux_x86_64
```

Comment:
Do we need these anymore? I don't think they're being referenced.

Reply:
We'll need those to select the platform we want to build for - i.e.: we'd be using these from the command line `./bazel build //main:sorbet --config=dbg-darwin`, which is why there aren't any explicit references in `.bazelrc`.

These are especially useful when cross compiling, since we need to select a platform that is different from the host.